rust/src/detect/byte_math.rs

```diff
@@ -17,8 +17,8 @@
 
 // Author: Jeff Lucovsky <jlucovsky@oisf.net>
 
-use crate::detect_parser::error::RuleParseError;
-use crate::detect_parser::parser::{parse_token, take_until_whitespace};
+use crate::detect::error::RuleParseError;
```

Comment:
I would have have put first the commit `Create detect module for rule parsing` and then the commit `detect/bytemath: convert parser to Rust` to avoid changing things twice in a PR.

But I know it is a pain so...

Reply:
And the commit message of `Create detect module for rule parsing` does not need to refer to ticket 5077 in my humble opinion as this is just code reordering, not adding the feature described in the ticket, which is done in commit `detect/bytemath: convert parser to Rust`